cron/photos.py

```diff
@@ -54,6 +55,33 @@ def download_photo(user):
         # Some students don't have photos if they never went for picture day
     return None
 
+    def download_photo(url):
+        try:
+            return download_photo_bytes(url)
+        except PIL.UnidentifiedImageError:
+            print("Unable to download {} with primary url")
+        try:
+            return download_photo_bytes(url)
+        except PIL.UnidentifiedImageError:
+            print("UNABLE to download " )
+            # If there is another issue 
+        return None
+
+    
+
+    def read_csv_as_dict():
+        photoids = {}
+        if 'idphotos_2021.csv':
+            with open('idphotos_2021.csv', newline='') as csvfile:
+                csvreader = csv.reader(csvfile, delimiter=' ', quotechar = '|')
+                for row in csvreader:
+                    url = row[4]
+                    urlparts = url.split("/")
+                    studentid = urlparts[6:8]

```

Comment:
you'll need to `.join` these I think to get the full id (and also convert to int)

Reply:
Do I need to say .join for 6, 7 and 8?

Reply:
student_id will be an array, you can then .join those to get a string.